src/main/java/org/cactoos/bytes/UncheckedBytes.java

```diff
@@ -44,7 +44,7 @@
     /**
      * Fallback.
      */
-    private final Func<Exception, byte[]> fallback;
+    private final Func<? super Exception, ? extends byte[]> fallback;
```

Comment:
@baudoliver7 `? extends byte[]` is needed?

Reply:
@andreoss Sorry, it isn't needed here.